ExposureNotification.App/ExposureNotification.App/App.xaml.cs

```diff
@@ -20,10 +21,18 @@ public App()
 			// Local Notification tap event listener
 			NotificationCenter.Current.NotificationTapped += OnNotificationTapped;
 
-			// Initialize the library which schedules background tasks, etc
+			// Initialize the library
 			Xamarin.ExposureNotifications.ExposureNotification.Init();
 
 			MainPage = new AppShell();
+
+			_ = InitializeBackgroundTasks();
+		}
+
+		async Task InitializeBackgroundTasks()
+		{
+			if (await Xamarin.ExposureNotifications.ExposureNotification.IsEnabledAsync())
+				await Xamarin.ExposureNotifications.ExposureNotification.ScheduleFetchAsync();
```

Comment:
The app now explicitly requests the fetch scheduling. This way, the app can decide when it wants to and after what checks.